setup.py

```diff
@@ -80,8 +84,8 @@
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
```

Comment:
I take it that 3.5 was already not supported, this is just a documentation update, right?

Reply:
Correct, Python 3.5 never was supported here anyway so these bits where the only things that where left from that commit. I could have changed the commit text to make it more clear that this is purely a doc change.